magento/magento2
# magento/community-features#238: Don't trigger reload of ui listing at page load.

### Description (*)

This PR is a re-creation of magento/magento2#26984.

### Related Pull Requests
https://github.com/magento/adobe-stock-integration/pull/1850


### Fixed Issues (if relevant)

1. Fixes magento/community-features#238: Don't trigger reload of ui listing at page load.

### Manual testing scenarios (*)

1. See magento/community-features#238: Don't trigger reload of ui listing at page load.

### Questions or comments


### Contribution checklist (*)
